app/src/pages/CardSubmission/WaitingForSubmissions.tsx

```diff
@@ -37,77 +37,78 @@ function WaitingForSubmissions() {
+                {waitingForPlayers.map((player) => (
+                  <>
+                    <PlayerChip username={player.username || ""} />
+                    <div style={{ width: 4 }} />
+                  </>
+                ))}
+              </Grid>
+              <Grid item>
+                <Typography variant="h5">
+                  {submittedSoFar}/{`${total} cards so far`}
+                </Typography>
+              </Grid>
+            </>
+          ),
+          [WaitingForSubmissionsState.SubmittedAssign]: (
             <Grid item>
-              {currentPlayer.role === PlayerRole.Host ? (
-                <Button
-                  size="large"
-                  variant="contained"
-                  color="primary"
-                  onClick={async () => {
-                    const players = teamsWithSequence(currentGame.players)
-                    await updateAllPlayers({
-                      variables: {
-                        gameId: currentGame.id,
-                        players: players.map(({ id, team, team_sequence }) => ({
-                          id,
-                          team,
-                          team_sequence,
-                        })),
-                      },
-                    })
-                    updateGameState({
-                      variables: {
-                        id: currentGame.id,
-                        state: GameStateEnum.TeamAssignment,
-                      },
-                    })
-                  }}
-                >
-                  Assign Teams
-                </Button>
-              ) : null}
+              All players submitted! As the host, you can now assign teams.
             </Grid>
-          )}
-        </>
-      ) : (
-        <>
-          <Grid item container justify="center">
-            Just waiting for everyone else...<div style={{ width: 4 }}></div>
-            {waitingForPlayers.map((player) => (
-              <>
-                <PlayerChip username={player.username || ""}></PlayerChip>
-                <div style={{ width: 4 }}></div>
-              </>
-            ))}
-          </Grid>
-          <Grid item>
-            <Typography variant="h5">
-              {submittedSoFar}/{`${total} cards so far`}
-            </Typography>
-          </Grid>
-          <Grid item>
-            <div style={{ marginTop: 50 }}>
-              <Fishbowl></Fishbowl>
-            </div>
+          ),
+          [WaitingForSubmissionsState.SubmittedWait]: (
+            <Grid item>
+              All players submitted, ${submittedSoFar} cards in total. Now we
+              are waiting on the host to start the game!
+            </Grid>
+          ),
+        }[state]
+      }
+      <Grid item container direction="column" spacing={2} alignItems="center">
+        {unscreenedCards.map((card, index) => (
+          <Grid item key={index}>
+            <BowlCard>{card.word}</BowlCard>
           </Grid>
```

Comment:
Next step: Add approve / rejection functionality and perhaps abstract this into a new component depending on how complex that gets?

Do you think the player's _name_ should be on the card as well?

Reply:
i think players name should be on card 👍 

and yea, new component sounds good. personally don't care if we c/p the card and buttons there